experimental/instrumentation-fetch/src/instrumentation.ts

```diff
@@ -0,0 +1,84 @@
+import { BaseInstrumentation, faro, globalObject } from '@grafana/faro-core';
+
+import { fetchGlobalObjectKey, originalFetch, originalFetchGlobalObjectKey } from './constants';
+
+export class FetchInstrumentation extends BaseInstrumentation {
+  readonly name = '@grafana/faro-web-sdk:instrumentation-fetch';
+  readonly version = '1.0.0'; // TODO - pull from package.json
+
+  initialize(): void {
+    console.log('Initializing FetchInstrumentation');
```

Comment:
Hint: 
I think the console.log is there for dev purposes.
if you want to log data later (debug, infor, warn, error) you can do this with Faros' internalLogger utility which is provided via the BaseInstrumentation.

```ts
this.internalLogger.info('foo')
```

Reply:
Yes this console log was here for debugging purposes, will switch it out in favor of the internal logger.